yang/frr-interface.yang

```diff
@@ -48,15 +69,49 @@ module frr-interface {
         description
           "Interface description.";
       }
-    }
-  }
 
-  typedef interface-ref {
-    type leafref {
-      require-instance false;
-      path "/frr-interface:lib/frr-interface:interface/frr-interface:name";
+      leaf if-index {
+        type int32 {
+          range "0..2147483647";
+        }
+        config false;
+        mandatory false;
+        description
+          "Interface index.";
+      }
+
+      leaf mtu {
+        type uint16 {
+          range "68..max";
+        }
+        config false;
+        description
+          "The size of the largest IPV4 packet that the interface
+           will send and receive.";
+      }
+
+      leaf speed {
+        type uint32;
+        config false;
+        description
+          "Interface speed.";
+      }
+
+      leaf type {
+        type identityref {
+          base ietf-if:interface-type;
+        }
+        config false;
+        description
+          "The link type of the interface.";
+      }
+
+      leaf phy-address {
+        type yang:mac-address;
+        config false;
+        description
+          "The interface's MAC address.";
+      }
```

Comment:
Missing state attributes from interface 

1. status
2. Flag
3. Metric
4. mtu6
5. configured

Reply:
Have added the most in the latest patch. status can be driven via `shutdown` rw field.